### ISSUE TITLE
Error when added UI / Squid / Local User - add linux-proxy as a user - will see error

### ISSUE THREAD

**Issue by ra-at-diladele-com** [diladele/websafety-issues#731]
```
NoReverseMatch at /squid/auth/localdb/user/list/

Reverse for 'ViewAuthLocalDbUsersUpdate' with arguments '('linux-proxy',)' not found. 
1 pattern(s) tried: ['squid/auth/localdb/user/(?P<pk>[\\w\\.]+)/edit/$']
```

**Comment by ra-at-diladele-com** [comment]
we must explicitly check for \w . and no  other symbols in form validate.

**Comment by ra-at-diladele-com** [comment]
fixed - need to test

**Comment by ra-at-diladele-com** [comment]
fixed; need a new build of 6.2
